org.openrgb.OpenRGB.yaml

```
@@ -0,0 +1,40 @@
+app-id: org.openrgb.OpenRGB
+runtime:  org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: openrgb
+finish-args:
+- --socket=x11
```

```suggestion
- --socket=x11
- --share=ipc
```

this is required for X11.

No wayland support?